userprofile/templates/userprofile/login.html

```diff
@@ -81,10 +81,12 @@ <h3>
 						{% csrf_token %}
 						{#{ form|bootstrap }#}
 						<div class="form-group">
-							<input class="form-control" type="text" id="id_username" name="username" maxlength="254" placeholder="{% trans 'Username' %}">
+							<label for="id_username">{% trans "Username" %}</label>
+							<input class="form-control" type="text" id="id_username" name="username" maxlength="254">
```

Comment:
These `id`s look a little strange, but I can see they've not been changed in this PR, so it's fine to leave them alone here!

Reply:
The form field id's could have a prefix related to the form: https://docs.djangoproject.com/en/3.0/ref/forms/api/#django.forms.Form.prefix

Not important for this page, but would make sense.